shared/serial_port_controller.py

```diff
@@ -188,6 +226,11 @@ def retrieve_setting(self, settings):
                 self.flow_control = 2
             case _:
                 self.flow_control = None
-    
+
+
+if __name__ == "__main__":
```

Comment:
This prints a lot of the test print statements to ensure code functionality. Will this be needed in the future state?